devtools/tsconfig.spec.json

```
@@ -1,6 +1,15 @@
 {
   "extends": "./tsconfig.json",
   "compilerOptions": {
+    "plugins": [
+      {
+        "name": "@bazel/tsetse",
+        "disabledRules": [
+          "must-type-assert-json-parse",
+          "must-use-promises"
```

Suggestion: I think it should be possible to use `void someUnusedPromise();` to silence this error at a specific line. Alternatively you could do `let _unusedProimse = someUnusedPromise();`.

I'd rather avoid disabling the rule for the whole project if we just need to work around it in one situation.